content/app/guidelines/wcag/_index.nb.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Tilgjengelighet
+description: God tilgjengelighet hjelper mennesker med funksjonshemninger å oppfatte innholdet vårt på en meningsfull måte. Ved å bruke Altinn 3 får du mye på kjøpet, men noen ting må du fortsatt huske på selv. 
+weight: 7
+weight: 30
+---
+
+Forskrift om universell utforming av IKT-løsninger stiller krav om at nettsider må oppfylle 35 av 61 suksesskriterier i standarden [Retningslinjer for tilgjengelig webinnhold (WCAG) 2.0](https://www.w3.org/Translations/WCAG20-no/). Sjekk ut [minimumskravene](https://www.uutilsynet.no/wcag-standarden/wcag-20-standarden/86) på UU-tilsynet som er omfattet av forskriften.
+
+
+
+
+## Sjekkliste
+
+- **Innholdsstruktur** <br>
```

Comment:
Stop using `<br>`!! Two spaces at the end of a line is the same in markdown. Same for most of the pages in this PR.